### DISCUSSION TITLE
Freeze Frame: always show reverse slider arrow

### DISCUSSION THREAD

**Discussion by sz-da** [ppy/osu#26718] [Ideas]
For some short sliders, the reverse arrow doesn't even show up. While it can be interesting trying to guess if a slider is a reverse slider or not, I think it's better to always show the reverse arrow for better readability.

https://github.com/ppy/osu/assets/76718358/99a7448e-8545-4953-945d-23f28ddc946e

**Comment by JamieWright99** [comment]
I agree with this. I guess it depends on what people expect from this mod but for me since sliders and circles are all faded in and readable at the same time it would make sense that reverse sliders are as well.
